drivers/emmc/emmc.c

```diff
@@ -0,0 +1,347 @@
+/*
+ * Defines a simple and generic interface to access eMMC device.
+ *
```

Comment:
Could you move this comment below the copyright/license text? I think it will defeat our static checkers.